engines/bops_api/app/jobs/bops_api/create_neighbour_boundary_geojson_job.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module BopsApi
+  class CreateNeighbourBoundaryGeojsonJob < ApplicationJob
+    queue_as :low_priority
+    discard_on ActiveJob::DeserializationError
+
+    def perform(planning_application)
+      response = ::Apis::OsPlaces::Query.new.find_addresses_by_radius(planning_application.lonlat.latitude, planning_application.lonlat.longitude).to_json
+
+      # API request fails
+      return if JSON.parse(response).empty?
+
+      results = JSON.parse(JSON.parse(response)["body"])["results"]
```

Comment:
Think we ought to add a `rescue JSON::ParserError` block to the method to handle any bad responses. Unless we want this to blow up but maybe we can trigger appsignal in that case